hybrid/HybridSampleApps/NoteSync/build.gradle

```diff
@@ -5,8 +5,9 @@ dependencies {
 }
 
 android {
-  compileSdkVersion 21
-  buildToolsVersion "21.1.1"
+  compileSdkVersion 23
+  buildToolsVersion '23.0.1'
+  useLibrary  'org.apache.http.legacy'
```

Comment:
Gotcha. I thought every project built on ```SalesforceSDK``` would automatically get this, since it's a dependency on the library's gradle file.